---
layout: default
---

This is the web page for the Next Generation EU project PRIN 2022 (2022Z4FZE9) project "Hunting metal ions within cryo-EM derived RNA structures",

The project started on 28-09-2023 and involved two units:
- A SISSA unit, lead by [Giovanni Bussi](https://scholar.google.com/citations?user=8qk_P2YAAAAJ)
- A CNR unit, lead by [Alessandra Magistrato](https://people.sissa.it/~alema/Introduction.html)

## About

RNA molecules perform a variety of key biological functions carrying genome information and acting as RNA-based catalysts (ribozymes or protein-directed ribozymes) to regulate gene expression, proteome diversification and, ultimately, to perform protein synthesis. RNA molecules can adopt extraordinarily complex folds, which are instrumental to conduct their functions. Due to their highly negative charge (each nucleobase carries a negative charge), the stability and the functionality of intricate RNA architectures is intimately connected with the presence of neutralizing monovalent and divalent counterions (Mg2+ and K+), which, besides conferring structural stability to their fold, may also aid catalysis.

The advent of single particle cryo-electron microscopy (Cryo-EM) techniques allowed the acquisition of structural information of complex protein and protein/RNA machines, and has nowadays started to resolve even the structures of highly flexible and conformationally heterogeneous RNA-only structures. These structures set the basis to greatly expand our mechanistic understanding of sophisticated RNA macromolecules. Nevertheless, in spite of the critical relevance of metal ions for their structural and functional properties of RNA machines, their assignment within cryo-EM maps is challenging, and, in most cases, limited to divalent cations. This severely hinders a characterization of the many different facets of RNA functions.

This project pushes the boundaries for a detailed atomic-level comprehension of RNA mechanisms by (i) developing cutting-edge computational methods and protocols based on classical and quantum-classical all-atom simulations to accurately assign mono and divalent cations within cryo-EM maps of RNA macromolecules and (ii) exploit these assignments to assess the function of biologically relevant RNA macromolecules. Namely, we plan to (i) develop an enhanced sampling method based on classical force fields and (ii) implement it in conjunction with state-of-the-art techniques to integrate molecular dynamics simulations and Cryo-EM maps, resulting in a procedure aimed to assign the position of Mg2+ and K+ metal ions within large RNA macromolecules. We will then (iii) assist, refine and validate the ions assignments by performing quantum-classical simulations and, ultimately, (iv) exploit the introduced methodologies to refine the structures of one of the largest RNA catalyst known to date, the group II intron ribozyme, and to unravel its catalytic mechanism. This information is key to rationally engineer the group II intron ribozyme as a genome editing tool.

## People

- Giovanni Bussi (PI, SISSA)
- Oliver Languin-Cattoën (Post-doc, SISSA)
- Elisa Posani (PhD student, SISSA)
- Alessandra Magistrato (PI, CNR)
- Pavel Janoš (Post-doc, CNR)

## Publications

- Ensemble Refinement of mismodeled cryo-EM RNA Structures Using All-Atom Simulations,
  Elisa Posani, Pavel Janoš, Daniel Haack, Navtej Toor, Massimiliano Bonomi, Alessandra Magistrato, Giovanni Bussi,
  [preprint](https://doi.org/10.1101/2024.07.24.604258)

